dispatch/BESLog.cc

```diff
@@ -76,10 +75,36 @@ const string BESLog::mark = string("|&|");
  * @see BESKeys
  */
 BESLog::BESLog() :
-    d_flushed(1), d_file_buffer(0), d_suspended(0), d_verbose(false), d_use_local_time(false), d_use_unix_time(false)
+    d_file_buffer(nullptr),
```

Comment:
I put this as a comment in the header because I looked at it first, but these should all be initialized in the class header.